armsrc/iso14443a.c

```diff
@@ -1183,7 +1188,15 @@ bool SimulateIso14443aInit(uint8_t tagType, uint16_t flags, uint8_t *data, tag_r
 
         // Configure the ATQA and SAK accordingly
         rATQA[0] &= 0xBF;
-        rSAKc1[0] = sak & 0xFB;
+
+        if(tagType == 11){
+            rSAKc1[0] = sak & 0xFC & 0X70;
+            DbpString(_YELLOW_("[ ") "Passando no Sak Penegui" _YELLOW_(" ]"));
+        }else{
+            rSAKc1[0] = sak & 0xFB;
+            //DbpString(_YELLOW_("[ ") "Passando no Sak Antigo" _YELLOW_(" ]"));
```

Comment:
remove this one too

Reply:
Sorry! I forgot to take it out of debug statement!